src/templates/season/list.pug

```diff
@@ -13,15 +13,13 @@ block content
       table.table.is-striped.is-middle-aligned.is-fullwidth
         thead
           tr
-            th Number
             th Name
             th Register
             if user && user.isAdmin
               th Edit
         tbody
           each season in seasons
             tr
-              td= season.number
```

Comment:
Any particular reason these were removed?

Reply:
I just saw no need for them since we already had name on there, keep it cleaner